bower.json

```diff
@@ -64,6 +64,11 @@
       "components/long/**/*.js",
       "components/bytebuffer/**/*.js",
       "components/protobuf/**/*.js"
+    ],
+    "libloki": [
+      "components/long/**/*.js",
```

Comment:
Was this one needed? 🤔

Reply:
I put this one back in because it's actually used by the ByteBuffer stuff